documentation/release_notes.rst

```diff
@@ -8,6 +8,64 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.3.0
+===============
+
+New Features
+------------
+- Added dynamic selection as an experimental feature. Provides functions such as ``select``,
+  ``submit`` and ``submit_and_wait``. It includes selection policies: ``fixed_resource_policy``,
+  ``round_robin_policy``, ``dynamic_load_policy``, ``auto_tune_policy``, and provides default
+  support for SYCL queues. 
+- Algorithms using vector execution policies are now vectorized whenever possible when the program is built with Intel® DPC++/C++ Compiler.
+- Added support for passing zip iterators as segment value data in ``reduce_by_segment``, ``exclusive_scan_by_segment``,
+  and ``inclusive_scan_by_segment``.
+- Improved performance of ``merge``, ``sort`` and ``stable_sort`` algorithms on GPU devices.
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``,
+  ``is_partitioned``, and ``lexicographical_compare`` algorithms with DPC++ execution policies.
+
+Fixed Issues
+------------
+- Fixed behavior of ``reduce_async`` function with ``binary_op`` parameter. 
+  Before it ignored the parameter and always used ``std::plus``.
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- When compiling with DPC++ compiler option, libstdc++ version 8 or libc++, and with ``-fsycl-pstl-offload`` option,
```

Comment:
Should we omit "option" in "DPC++ compiler option"? Does it rather imply `-fsycl` option? @kboyarinov, could you help?

Reply:
"option" word should be definitely removed. Regarding the `-fsycl` option - `-fsycl-pstl-offload` option requires to use `-fsycl` as well. I don't think that `-fsycl` should be mentioned.

Reply:
Agreed. Please change it to "When compiling with DPC++ compiler, libstdc++ version 8 or libc++, and with ``-fsycl-pstl-offload`` option

Reply:
Added the suggested